### PR TITLE
Iss383 sidebar

### PR DESCRIPTION
**Pull request by tylercrosse**
Currently lacks tests, but getting closer to being able to merge. I would love some feedback :eyes:

### CODE REVIEW

File: app/views/cohorts/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<nav class="page-nav navbar-relative">
```
Comment (nolds9) [inline app/views/cohorts/show.html.erb]: I think a style guide we are trying to maintain is that any classes that are used for javascript selection should be prefixed with `js`, i.e. `js-page-nav`

**Comment by nolds9** [comment]
:+1: Really cool Tyler! Some initial ux thoughts: it would be great if we could consolidate the navigation so there's not 3 separate nav bar's. Also possibly we be cool to have a way to "toggle" the navigation, that is add an icon and functionality to expand and collapse it. But these are all nice to haves, would be happy to have it in as is.

**Comment by tylercrosse** [comment]
I think this is good enough to be useful, would love it to be looked over and merged in.

File: app/views/cohorts/show.html.erb (hunk 1 above)
Comment (andrewsunglaekim) [inline app/views/cohorts/show.html.erb]: what if its for both CSS && JS? Might be ok

**Comment by andrewsunglaekim** [comment]
:+1:

**Pull request merged**